assets/css/src/amp-admin.css

```diff
@@ -35,12 +46,6 @@
 	width: 15px;
 }
 
-.theme-browser .theme {
-	float: none;
-	display: inline-block;
-	vertical-align: top;
-}
-
```

Comment:
This style rule didn't seem to be doing anything anymore. I think it was needed for the AMP-compatible banner which we replaced with a badge. @dhaval-parekh is that right?

Reply:
Yes, This style is no longer needed. It was added for the AMP-compatible banner.